test.js

```diff
@@ -88,6 +88,16 @@ describe('testData', function () {
         assert.strictEqual(testRuns, 2);
     });
 
+    it('relays the return value of the test function', function () {
+        var tests = testData(1).test('example test', function () {
+            return 1337;
```

Comment:
Funny, though we have more of a soft spot for [42](https://en.wikipedia.org/wiki/42_(number)#The_Hitchhiker.27s_Guide_to_the_Galaxy)... :wink:

(Just for fun, try to convert @101100 's name from binary :stuck_out_tongue_winking_eye: )